src/server/status/status.js

```diff
@@ -3,22 +3,21 @@ import states from './states';
 import { EventEmitter } from 'events';
 
 class Status extends EventEmitter {
-  constructor(name, server) {
+  constructor(plugin, server) {
     super();
 
-    this.name = name;
+    this.plugin = plugin;
     this.since = new Date();
     this.state = 'uninitialized';
     this.message = 'uninitialized';
 
     this.on('change', function (previous, previousMsg) {
       this.since = new Date();
-      let tags = ['status', name];
+      let tags = ['status', 'plugin:' + this.plugin.toString()];
       tags.push(this.state === 'red' ? 'error' : 'info');
```

Comment:
Oops, the added space was unintentional. Mark it up to hard-to-read string concatenation.